app/Actions/Photo/Delete.php

```diff
@@ -2,23 +2,32 @@
 
 namespace App\Actions\Photo;
 
+use App\Exceptions\ModelDBException;
 use App\Models\Photo;
 
 class Delete
 {
+	/**
+	 * @throws ModelDBException
+	 */
 	public function do(array $photoIds): void
```

Comment:
The answer is simple: This action is not used anymore but dead code. So my IDE did not catch it, when I re-factored the code. This action is one of the unnecessary ones [as discussed in this comment](https://github.com/LycheeOrg/Lychee/pull/1217#discussion_r825428655).

Probably, I should simply remove all of these unnecessary actions so that nobody is wondering anymore.

Reply:
Resolved by simple deletion of offending class.